ui/src/store/dashboard.ts

```diff
@@ -47,12 +48,12 @@ export default class DashboardStore {
     return this.viewIndexSubject;
   }
 
-  static setTopicParam(value: TopicParam) {
-    this.topicParamSubject.next(value);
+  static setTopicOrProjectParam(value: TopicParam | ProjectParam) {
```

Comment:
A priori it is OK to use a single observable for topic and project since they are exclusive.
To make it clearer I would wrap the values:
{
  kind: 'project',
  param: ProjectParam
} / 
{
  kind: 'topic',
  param: TopicParam
}